src/components/pageLayout/pageHeader.js

```diff
@@ -1,35 +1,60 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { withTranslation } from 'react-i18next';
 import {
   PageHeader as RcsPageHeader,
   PageHeaderTitle
 } from '@redhat-cloud-services/frontend-components/components/cjs/PageHeader';
+import { Button } from '@patternfly/react-core';
+import { ExternalLinkSquareAltIcon } from '@patternfly/react-icons';
+import { translate } from '../i18n/i18n';
```

Comment:
#### Required:
Remove translate and use the `t` prop. Using `withTranslate` automagically adds the `t` prop. You should be able to emulate other components with `t`

#### History:
It's subtle and confusing, `translate` gets around the need to be a full fledged component, i.e. support functions.

Reply:
Appears I am mistaken, good catch @zanewoodfin ! 

Looks like it'll need to be `translate`. Double check to see if you can remove `withTranslation` at the bottom. 

We'll open a new issue to see if we can focus the "libraries" for "locale"

Reply:
see #354